instrumentation/kafka-clients/src/main/java/brave/kafka/clients/TracingConsumer.java

```diff
@@ -254,6 +255,11 @@ public Map<TopicPartition, OffsetAndMetadata> committed(
     return delegate.committed(partitions, timeout);
   }
 
+  // Do not use @Override annotation to avoid compatibility issue version < 3.7
```

Comment:
this is how we address new functions added, while still supporting running with older versions